internal/cmd/kafka/command_cluster.go

```diff
@@ -85,3 +88,16 @@ func (c *clusterCommand) validArgs(cmd *cobra.Command, args []string) []string {
 	}
 	return suggestions
 }
+
+func convertClusterNameToId(input string, environmentId string, v2Client *ccloudv2.Client) (string, error) {
```

Comment:
```suggestion
func (c *command) convertClusterNameToId(name, environmentId string) (string, error) {
```

Reply:
I think I have to keep it the header this way to make it reusable for the `--cluster` flag, which I'll handle in dynamic_context.go. I think I might move this function to `pkg/utils` since it might result in a circular import